app/views/mail/register.scala.html

```diff
@@ -1,5 +1,62 @@
-@( name: String, brainDBresult: Option[String], enableAutoVerify: Boolean ) Hello @name, thank you
-for signing up for an webKnossos account! @if(!enableAutoVerify) { Your account is going to get
-activated after an administrator has reviewed your details. } @if(brainDBresult.isDefined) {
-@brainDBresult.get } Enjoy using webKnossos Your webKnossos Team
+@(name: String, brainDBresult: Option[String], enableAutoVerify: Boolean ) 
 
+@emailBaseTemplate() {
+
+  <p>Hi @name,</p>
+
+  @if(!enableAutoVerify) { 
+    <p>
+      welcome to your new webKnossos account. You are almost ready to start exploring and to collaboratively
+      annotate large 3D datasets.
+    </p>
+    <p>Your account is currently under review. An administrator will activate you soon.</p>
+  } else {
+    <p>
+      welcome to your new webKnossos account. Get ready to start exploring and to collaboratively
+      annotate large 3D datasets.
+    </p>
+  }
+
+  @if(brainDBresult.isDefined) { 
+    <p>@brainDBresult.get</p>
+  } 
+
+  <h2 style="font-size: 1rem;">Get started</h2>
+  <ul style="font-size: 1rem;">
+    <li>
+      Use webKnossos for free with any of the public datasets on
+      <a href="https://webknossos.org">webknossos.org</a>. All webKnossos features can be used
+      with your new free account.
+    </li>
```

Comment:
I'm not sure about this section. Wouldn't we direct users away from the webKnossos instance they just signed up for to the webknossos.org instance. I fear that this might cause confusion. The next list item is also somewhat directed at webknossos.org.